### PR TITLE
Updated README.md for making it easier getting started

### PR DESCRIPTION
**Pull request by maxadorable**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,91 @@
...
+  <a href="https://github.com/formulaslug/fs-2-schematics">
+    <img src="docs\FS-01-high-resolution.png" alt="Logo" width="100" height="60">
+  </a>
+  <h3 align="center">FS-2 Schematics</h3>
 
-This is the electrical schematics repository for the FS-2 racecar.
+  <p align="center">
+    This is the electrical schematics repository for the FS-2 racecar.
+    <br />
+    <a href="https://github.com/othneildrew/Best-README-Template/issues">Report Bug</a>
```
Comment (colescott) [inline README.md]: Looks like these links need modifying to link to this repo

File: README.md [hunk 2]
```diff
@@ -1,11 +1,91 @@
...
+    ·
+    <a href="https://github.com/othneildrew/Best-README-Template/issues">Request Feature</a>
```
Comment (colescott) [inline README.md]: This one too

File: README.md [hunk 3]
```diff
@@ -1,11 +1,91 @@
...
+  </p>
+</p>
 
-## Setup
+
+## Quick Start
```
Comment (colescott) [inline README.md]: Maybe we can remove this section and just have the "Getting started" one

**Review by colescott — CHANGES_REQUESTED** [review]
Looks good. Might want to fix the links then we can merge

**Review by maxadorable — COMMENTED** [review]
Fixed the links

**Review by colescott — APPROVED** [review]
lgtm! thanks

**Pull request merged**